portfolio/src/main/webapp/index.html

```diff
@@ -49,53 +50,148 @@ <h4>Software Engineering Intern</h4>
     </div>
 
     <!-- Main Content Section -->
+
     <div id="home" class="menu">
-      <section>
-          <div class="info">
-            <div class="title"><h2>About Me</h2></div>
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-              </p>
+      <section id="home" class="landing">
+        <div class="content"> 
+          <h2>Hi there.<br> I'm <span> Kayla Leung </span></h2>
+          <button id="testbutton" class="button" onclick="document.location = '#aboutme'">About Me</button>
+        </div>
+      </section>
+    </div>
+
+    <div id="aboutme" class="menu">
+      <section class="banner">
+        <div class="info">
+          <div class="title">
```

Comment:
You don't need a div wrapper here - put your CSS directly on the h1!

Reply:
Also, think of your headings in a hierarchal way, like the outline of an article (that's how screen readers and search engine robots like Google think of it). Right now you have a h2 above which is the "Hi there I'm Kayla Leung" but here you have a h1 "About Me" so the hierarchy is a bit inverted.